SYCL/ESIMD/lsc/lsc_load2d_compare.cpp

```diff
@@ -0,0 +1,103 @@
+//==------------ load2d_compare.cpp - DPC++ ESIMD on-device test ----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
```

Comment:
requiring pvc and emulator eliminates the need in this 'unsupported: cuda || hip'
```suggestion
```